internal/cli/user_blocks.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5/management"
+)
+
+var (
+	userID = Argument{
+		Name: "user_id",
+		Help: "user_id of the user.",
+	}
+)
+
+func userBlocksCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "user-blocks",
+		Short: "Manage brute-force protection user blocks.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUserBlocksByUserIdCmd(cli))
+	cmd.AddCommand(deleteUserBlocksByUserIdCmd(cli))
+	return cmd
+}
+
+func listUserBlocksByUserIdCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		user_id string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "listByUserId",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List user-blocks by user_id",
```

Comment:
Please do not include the API2 entities as-is, instead describe what the command does in the context of a single user.

Reply:
Assume the user knows nothing about API2.